### ISSUE TITLE
14.C Layout Table, it says <table role="presentation"> will pass the test

### ISSUE THREAD

**Issue by sanakapasi** [Section508Coordinators/DHSA-TT-200-A#477]
Under Vides Demo > 14.C Layout Table

This video says <table role="presentation"> will pass the accessibility test under PASS Criteria. However, within the same video it  says <table role="presentation"> will fail the test.

Which one is correct? Please assist. 

Best,
Sana Kapasi
